l10n-pocket/en/pocket/platforms.ftl

```diff
@@ -2,35 +2,40 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-### iOS URL: https://www-dev-pocket.allizom.org/ios/
-### iOS URL: https://www-dev-pocket.allizom.org/android/
-### iOS URL: https://www-dev-pocket.allizom.org/safari/
-### iOS URL: https://www-dev-pocket.allizom.org/chrome/
-### iOS URL: https://www-dev-pocket.allizom.org/opera/
-### iOS URL: https://www-dev-pocket.allizom.org/edge/
-### iOS URL: https://www-dev-pocket.allizom.org/welcome/
-
-
-pocket-platform-get-app = Get the app
-pocket-platform-install = Install
-pocket-platform-log-in = Log in
+## iOS URL: https://dev.tekcopteg.com/ios/
 
 pocket-platform-ios-title = { -brand-name-pocket } for { -brand-name-ios }
-pocket-platform-ios-view-list = View your list on your iPhone or iPad. Download the free { -brand-name-pocket } app for { -brand-name-ios } from the { -brand-name-apple-app-store }.
+pocket-platform-ios-view-list = View your list on your { -brand-name-iphone } or { -brand-name-ipad }. Download the free { -brand-name-pocket } app for { -brand-name-ios } from the { -brand-name-apple-app-store }.
+
+## Android URL: https://dev.tekcopteg.com/android/
 
 pocket-platform-android-title = { -brand-name-pocket } for { -brand-name-android }
 pocket-platform-android-view-list = View your list on your { -brand-name-android} device. Download the free { -brand-name-pocket } app from { -brand-name-google-play }.
 
+## Chrome URL: https://dev.tekcopteg.com/chrome/
 
 pocket-platform-chrome-title = { -brand-name-pocket } for { -brand-name-chrome}
 
+## Safari URL: https://dev.tekcopteg.com/safari/
+
 pocket-platform-safari-title = { -brand-name-pocket } for { -brand-name-safari}
 
+## Opera URL: https://dev.tekcopteg.com/opera/
+
 pocket-platform-opera-title = { -brand-name-pocket } for { -brand-name-opera }
 
+## Edge URL: https://dev.tekcopteg.com/edge/
+
 pocket-platform-edge-title = { -brand-name-pocket } for { -brand-name-edge }
 
-pocket-platform-browser-installing = Installing the { -brand-name-pocket } browser extension installs buttons that let you save items with one click.
+## Welcome URL: https://dev.tekcopteg.com/welcome/
 
 pocket-platform-welcome-title = { -brand-name-pocket } for Your Browser
 pocket-platform-welcome-when-browsing = When browsing the web, simply push this button in your Bookmarks Bar to add pages to your { -brand-name-pocket } list.
+
+##
```

Comment:
Also, referencing the [Fluent docs for comments](https://projectfluent.org/fluent/guide/comments.html), I wanted to place the rest of the other strings (that don't adhere to one specific URL) at the bottom of the file, and I used `##` to essentially "escape" the previous group comment.